crates/ra_syntax/src/ast.rs

```diff
@@ -217,7 +218,7 @@ fn test_doc_comment_multi_line_block_strips_suffix() {
 fn test_comments_preserve_trailing_whitespace() {
     let file = SourceFile::parse(
         r#"
-/// Representation of a Realm.   
+/// Representation of a Realm.
```

Comment:
the trailing whitespace here was intentional and is being tested for :sweat_smile:

Reply:
My editor deleted it :man_facepalming: